WordPress/Classes/Models/Post.swift

```diff
@@ -195,7 +195,7 @@ class Post: AbstractPost {
 
         if let originalPost = original as? Post {
 
-            if tags?.characters.count != originalPost.tags?.characters.count && tags != originalPost.tags {
+            if tags ?? "" != originalPost.tags ?? "" {
```

Comment:
To be completely fair @elibud, I think it'd be better for us to continue the discussion outside of this PR... I have a few thoughts on changes we could make, but they definitely fall outside of the scope of your fix.